src/search/product.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * The structure of a product.
+ */
+export interface Product extends Record<string, any> {
+  /** The name of this product. */
+  title?: string;
+  /** The display price of this product. */
+  price?: number;
+  /** The URL of this product's image. */
+  imageSrc?: string;
+  /** The alternative text for this product's image. */
+  imageAlt?: string;
+  /** The URL for product details page. */
```

Comment:
'The URL for _this product's_ details page' to be consistent with the rest of the Product interface doc comments?

Reply:
Updated.